forge-gui/res/cardsfolder/upcoming/long_list_of_the_ents.txt

```diff
@@ -0,0 +1,11 @@
+Name:Long List of the Ents
+ManaCost:G
+Types:Enchantment Saga
+K:Saga:6:DBNoteType,DBNoteType,DBNoteType,DBNoteType,DBNoteType,DBNoteType,
+SVar:DBNoteType:DB$ ChooseType | Type$ Creature | Note$ True | SpellDescription$ Note a creature type that hasn't been noted for CARDNAME. When you cast your next creature spell of that type this turn, that creature enters the battlefield with an additional +1/+1 counter on it.
+T:Mode$ SpellCast | ValidCard$ Creature.NotedType | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Secondary$ True | Execute$ TrigEffect | TriggerDescription$ When you cast your next creature spell of that type this turn, that creature enters the battlefield with an additional +1/+1 counter on it.
```

Comment:
delayed trigger

Reply:
your problem is all chapters will trigger for every noted type

Reply:
Will this need engine changes or are there similar cards I should look at ?

Reply:
You might need to convert it into an Effect so you'll have another host you can transfer the type directly

Reply:
hmm you mean putting the choosetype into an effect ? how will it remember the "hasn't been noted" part ?

Reply:
I'll skip this one for now